src/services/sessions.js

```diff
@@ -216,17 +331,32 @@ module.exports = class SessionsHelper {
 
 	static async update(sessionId, bodyData, userId, method, orgId) {
 		let isSessionReschedule = false
+		let isSessionCreatedByManager = false
 		try {
-			let mentorExtension = await mentorExtensionQueries.getMentorExtension(userId)
-			if (!mentorExtension) {
-				return common.failureResponse({
-					message: 'INVALID_PERMISSION',
-					statusCode: httpStatusCode.bad_request,
-					responseCode: 'CLIENT_ERROR',
-				})
+			// To determine the session is created by manager or mentor we need to fetc the session details first
+			// Then compare mentor_id and created_by information
+			// If manager is the session creator then no need to check Mentor extension data
+			const sessionDetail = await sessionQueries.findById(sessionId)
+			if (
+				sessionDetail.metor_id &&
+				sessionDetail.created_by &&
+				sessionDetail.metor_id !== sessionDetail.created_by
+			) {
+				isSessionCreatedByManager = true
 			}
 
-			const sessionDetail = await sessionQueries.findById(sessionId)
+			if (!isSessionCreatedByManager) {
```

Comment:
@VISHNUDAS-tunerlabs If the session is created by the session manager then, the mentor can't update that session

Reply:
@rakeshSgr ok. got your point

Reply:
@rakeshSgr ok